src/Avalonia.Controls/NumericUpDown/NumericUpDown.cs

```diff
@@ -370,6 +370,20 @@ protected override void OnKeyDown(KeyEventArgs e)
             }
         }
 
+        /// <summary>
+        /// Called to update the validation state for properties for which data validation is
+        /// enabled.
+        /// </summary>
+        /// <param name="property">The property.</param>
+        /// <param name="value">The new binding value for the property.</param>
+        protected override void UpdateDataValidation<T>(AvaloniaProperty<T> property, BindingValue<T> value)
```

Comment:
Just curious, why isn't `/// <inheritdoc/> used instead of copying the comments? This should only be documented in the base class to avoid errors over time. Visual Studio supports this now (not sure about VS for Mac)

Reply:
Just a copy and paste from AutoCompleteBox

Reply:
@robloo good point. I don't remember from which version of VS19 it works though, it was added somewhere in 2019-2020.
Also it's possible to add ```/// <inheritdoc/>``` on the class and all docs will be inherited automatically.
It's not a big deal for this PR though.

Reply:
Yea, I just noticed it here and was curious about the Avalonia convention. Definitely recommend to start using that in practice.